mmv1/templates/terraform/examples/cloud_run_service_pubsub.tf.erb

```diff
@@ -0,0 +1,62 @@
+# [START cloud_run_service_pubsub_service]
+resource "<%= ctx[:vars]['primary_resource_type'] %>" "<%= ctx[:primary_resource_id] %>" {
+    name     = "<%= ctx[:vars]['cloud_run_service_name'] %>"
+    location = "us-central1"
+    template {
+      spec {
+            containers {
+                image = "gcr.io/cloudrun/hello"
+            }
+      }
+    }
+    traffic {
+      percent         = 100
+      latest_revision = true
+    }
+}
+# [END cloud_run_service_pubsub_service]
+
+# [START cloud_run_service_pubsub_sa]
+resource "google_service_account" "sa" {
+  account_id   = "cloud-run-pubsub-invoker"
+  display_name = "Cloud Run Pub/Sub Invoker"
+}
+# [END cloud_run_service_pubsub_sa]
+
+# [START cloud_run_service_pubsub_run_invoke_permissions]
+resource "google_cloud_run_service_iam_binding" "binding" {
+  location = google_cloud_run_service.<%= ctx[:primary_resource_id] %>.location
+  service = google_cloud_run_service.<%= ctx[:primary_resource_id] %>.name
+  role = "roles/run.invoker"
+  members = ["serviceAccount:${google_service_account.sa.email}"]
+}
+# [END cloud_run_service_pubsub_run_invoke_permissions]
+
+# [START cloud_run_service_pubsub_token_permissions]
+resource "google_project_iam_binding" "project" {
+  role    = "roles/iam.serviceAccountTokenCreator"
+  members = ["serviceAccount:${google_service_account.sa.email}"]
+}
+# [END cloud_run_service_pubsub_token_permissions]
+
+# [START cloud_run_service_pubsub_topic]
+resource "google_pubsub_topic" "topic" {
+  name = "<%= ctx[:vars]['pubsub_topic'] %>"
+}
+# [END cloud_run_service_pubsub_topic]
+
+# [START cloud_run_service_pubsub_sub]
+resource "google_pubsub_subscription" "subscription" {
+  name  = "<%= ctx[:vars]['pubsub_subscription'] %>"
+  topic = google_pubsub_topic.<%= ctx[:vars]['pubsub_topic'] %>.name
+  push_config {
+    push_endpoint = google_cloud_run_service.<%= ctx[:primary_resource_id] %>.status[0].url
```

Comment:
It looks like `status` is not a list, so I think that it should be safe to remove the `[0]` (and I'm not entirely sure whether it would work to leave it in)

```suggestion
    push_endpoint = google_cloud_run_service.<%= ctx[:primary_resource_id] %>.status.url
```